### ISSUE TITLE
[MAS4.3.1] Links appearing after activating the 'Show Markdown using Adaptive cards' button are not able to adopt the high contrast theme.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1343]
**Actual Result:**
Links appearing after activating the 'Show Markdown using Adaptive cards' button are not able to adopt the high contrast theme.

**Note:** Same issue repro with High contrast white mode.

**Expected Result:**
Links should be able to adopt the high contrast modes. 

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Content will upload, Navigate to any "Show Markdown using Adaptive cards button" button by using Tab key and press Enter on it.
5. New data will upload.
6. Go to Settings-> Ease of Access-> High Contrast Settings-> Apply different High Contrast black Mode.
7. Observe that Focus indicator is not clearly visible on 'Date and Time' Flyout's elements.

**User Impact:**
Any images, icons, hyperlinks, text or visual focus disappear when high contrast mode is enabled, make that content unusable/invisible to the user. 

**MAS Reference:**
[MAS 4.3.1 - No Disruption of Accessibility Features (42A)](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74})

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Theme:** High Contrast

**Attachment:**
![mas4 3 1_links does not adopt the high contrast theme](https://user-images.githubusercontent.com/41668935/48412334-5bf2f980-e76a-11e8-8ba7-14038ed6e079.png)

**Comment by corinagum** [comment]
@priysn It appears that the links you have indicated in your screenshot are actually visited links, which is why they are displaying as green. Visiting other websites in high contrast mode reflect this behavior, where visited links become green after click.

**Comment by compulim** [comment]
We observe the same behavior (yellow on unvisited, green on visited link) when we are constructing a standard HTML5 `<a>` and showing it on Edge. We used the sample here, https://www.w3schools.com/html/tryit.asp?filename=tryhtml_links_w3schools.

**Comment by corinagum** [comment]
After discussing with Accessibility, we have confirmed that this is an OS feature, not a Web Chat bug.

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.3.1
